crates/ruff/resources/test/fixtures/flake8_pyi/PYI026.pyi

```diff
@@ -0,0 +1,18 @@
+import typing
+from typing import TypeAlias, Literal, Any
+
+NewAny = Any
+OptinalStr = typing.Optional[str]
+Foo = Literal["foo"]
+IntOrStr = int | str
+AliasNone = None
+
+NewAny: typing.TypeAlias = Any
+OptinalStr: TypeAlias = typing.Optional[str]
+Foo: typing.TypeAlias = Literal["foo"]
+IntOrStr: TypeAlias = int | str
+AliasNone: typing.TypeAlias = None
+
```

Comment:
Perhaps a test case with another existing annotation would be good e.g. `IntOrStr: Foo = int | str`